bootstrap/cmd/bootstrap/app/server.go

```diff
@@ -454,5 +460,28 @@ func Run(opt *options.ServerOption) error {
 	log.Infof("App root %v", kfApp.Root())
 
 	fmt.Printf("Initialized app %v\n", opt.AppDir)
+
+	if opt.Apply {
+		log.Infof("Apply kubeflow components...")
+		rawCmd := "ks show default | kubectl apply -f -"
```

Comment:
why not use API? There will be all sorts of issues with wrapping CLI, most notably error handling.

Reply:
Haven't figured out a elegant way:
1. ks runApply API expects clientcmd.ClientConfig, which kind of have soft dependency on existence of ```~/.kube/config```
2. If use k8s client-go API, would be quite verbose if we create all resources one by one.